### PR TITLE
[*] CORE: PHPDoc - inline docs for local variables

### PR DESCRIPTION
**Pull request by gskema**

### CODE REVIEW

File: controllers/admin/AdminAddressesController.php [hunk 1]
```diff
@@ -26,6 +26,9 @@
 
 class AdminAddressesControllerCore extends AdminController
 {
+	/** @var Address */
+	protected $object;
```
Comment (jeromenadaud) [inline controllers/admin/AdminAddressesController.php]: Hi, $object variable is already declared in AdminController as protected.

File: controllers/admin/AdminCmsController.php [hunk 2]
```diff
@@ -26,6 +26,9 @@
 
 class AdminCmsControllerCore extends AdminController
 {
+	/** @var CMS */
+	protected $object;
```
Comment (jeromenadaud) [inline controllers/admin/AdminCmsController.php]: Hi, $object variable is already declared in AdminController as protected.

File: controllers/admin/AdminCustomersController.php [hunk 3]
```diff
@@ -26,6 +26,9 @@
 
 class AdminCustomersControllerCore extends AdminController
 {
+	/** @var Customer */
+	protected $object;
```
Comment (jeromenadaud) [inline controllers/admin/AdminCustomersController.php]: Hi, $object variable is already declared in AdminController as protected.

**Comment by gskema** [comment]
Hello,
there may a be a couple more re-declaration like this from previous commits;

There may be another way to attach PHPDoc to this variable:
```
/**
 * Class AdminSomeObjectCore
 *
 * @property SomeObject $object
 */
```

This way is usually used for magic properties. Let me know which one you prefer. I'll then unify all PHPDoc for `$object` member

**Comment by jeromenadaud** [comment]
Hi,

I understand you reason, but please remove $object re-declaration on AdminController extended classes.
As in php doc we can reffer to class by namespace like \PrestaShop\AdminController. 
We will add soon namespace, it will be more easy for you to get magic properties.

Best regards.

**Comment by gskema** [comment]
So, shall I remove all re-declared properties in AdminControllers (only `$object`) and add `@property` in class PHPDocs instead?

**Comment by jeromenadaud** [comment]
Yes please.

Thank you very much for your contribution.

Best regards.

**Comment by gskema** [comment]
Pull request updated, remove re-declared files and added phpdoc for admin controllers.

**Comment by jeromenadaud** [comment]
Hi,

Great job @gskema !
Thanks.

Best regards

**Pull request merged**